src/dotnet-scaffolding/dotnet-scaffold/dotnet-scaffold.csproj

```diff
@@ -8,19 +8,30 @@
     <PackageTags>dotnet;scaffold</PackageTags>
     <PackageId>Microsoft.dotnet-scaffold</PackageId>
     <RootNamespace>Microsoft.DotNet.Tools.Scaffold</RootNamespace>
+    <ImplicitUsings>enable</ImplicitUsings>
     <NoWarn>$(NoWarn);8002</NoWarn>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\Microsoft.DotNet.Scaffolding.Core\Logging\CleanConsoleFormatter.cs" Link="Logging\CleanConsoleFormatter.cs" />
```

Comment:
Is this for the interim, wondering if this will complicate things in the future as these expand?

Reply:
Yeah, sorry, should have clarified that. The only thing "shared" between `dotnet-scaffold` and `.Core` is the `ComponentModel` library, but that's not the right place for this. I made it a shared file for the moment to get around that (since we don't need binary compat) but when we have further discussions about what exactly our package/library layout looks like we'll need to discuss where something might live that is shared between these layers.

Reply:
I added a comment to the .csproj file to hopefully remind us to change that

Reply:
cool sgsg